packages/dashboard/README.md

```diff
@@ -8,109 +8,26 @@ Follow the build instructions [here](../../README.md/#launching).
-* `dispensers`, where each dispenser can define its own
-  * `icons`, path to the icon image
-  * `location`, x, y, yaw, and level
-* `logos`, where named logos can be defined, most prominently `headerLogo` for the dashboard, default to Open-RMF logo
-* `helpLink`, changes the link of the help button on the `appbar`, defaults to the [`ros2multirobotbook`](https://osrf.github.io/ros2multirobotbook/rmf-core.html)
-* `reportIssue`, changes the link of the issue button on the `appbar`, defaults to [rmf-web issues](https://github.com/open-rmf/rmf-web/issues)
-* `defaultZoom`, the default zoom level of the map whenever the page is refreshed and the map is rendered, default value as `5` (the higher the value, the larger the entities appear in the map view)
-* `defaultRobotZoom`, the zoom level when the map's view is centered on a robot, whenever a task row or robot row is clicked, default value as `40` (the higher the value, the larger the robot icon appears in the map view)
-* `attributionPrefix`, the attribution text on the right bottom corner of the map, default as `OSRF`
-* `loggedInDisplayLevel`, the name of the map level to be displayed whenever a user logs in or the page is refreshed, defaults to the first level in the building map
-* `customTabs`, whether to display and use custom tabs for customized layouts, defaults to false
-* `adminTab`, whether to display the admin tab (only accessible by admins), to manage groups and permissions, defaults to false
-
-Custom fields can also be easily added and used in the dashboard, some examples,
-* `pickupZones`, this is a custom field implemented to allow a custom list of pickup zones to be used in a task form dropdown list
-* `cartIds`, this is a custom field implemented to allow a custom list of cart IDs to be used in a task form dropdown list
-
-An example directory structure can be found in `rmf_demos`, under the branch [`rmf-web-dashboard-resources`](https://github.com/open-rmf/rmf_demos/tree/rmf-web-dashboard-resources/rmf_demos_dashboard_resources/office)
-
-Here is another example `main.json`,
+The `resources` option in the app config allows configuring various assets used by the dashboard. At the moment, it mainly allows configuring the header logo and robot icons, they should point to either the full url or absolute url, for example
 
 ```json
-{
-  "robots": {
-    "tinyRobot": {
-      "icons": {
-        "tinyRobot": "/robots/tinyRobot/tinyRobot.png",
-        "model1": "/robots/tinyRobot/model1.png"
-      },
-      "scale": 0.004
-    }
-  },
-  "dispensers": {
-    "coke_dispenser": {
-      "icons":{
-        "coke_dispenser": "/icons/fridge.png"
-      },
-      "location": {
-        "x":16.85,
-        "y":-4.8,
-        "yaw":0,
-        "level_name":"L1"
-      }
-    },
-  },
-  "logos":{
-    "headerLogo":{
-      "icons":{
-        "headerLogo":"/icons/headerLogo.png"
+  "resources": {
+    "default": {
+      "fleets": {},
```

Comment:
```suggestion
      "fleets": {
        "tinyRobot": {
          "default": {
            "icon": "/resources/tinyRobot.png",
            "scale": 0.00123
          }
        }
      },
```

Reply:
or perhaps point it directly to https://raw.githubusercontent.com/open-rmf/rmf_demos/rmf-web-dashboard-resources/rmf_demos_dashboard_resources/office/icons/tinyRobot.png

Reply:
pointing to github is a great idea!